### PR TITLE
npm run serve -> npm run start

### PR DESCRIPTION
**Pull request by sheerun**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -107,13 +107,13 @@ For a thorough explanation of the testing procedure, see the
 
 #### Performance testing
 
-With the production server running (i.e. while `$ npm run serve` is running in
+With the production server running (i.e. while `$ npm run start` is running in
```
Comment (mxstbr) [inline docs/README.md]: This should be `$ npm run start:production`!

**Comment by mxstbr** [comment]
Very good catch @sheerun! One slight change and then we'll merge this – thanks so much for taking the time to fix this!

**Comment by sheerun** [comment]
I wish one could edit PRs from web ;(

**Comment by sheerun** [comment]
ok, done

**Comment by mxstbr** [comment]
Thanks!

**Pull request merged**